src/core/library.c

```diff
@@ -1802,6 +1803,8 @@ MsQuicOpenVersion(
 
     Api->DatagramSend = MsQuicDatagramSend;
 
+    Api->ConnectionPoolCreate = MsQuicConnectionPoolCreate;
```

Comment:
This needs to be wrapped in the preview features flag

Reply:
Well it builds fine when preview features are disabled, so I guess I don't need that